src/main/resources/avro/alleleAnnotations.avdl

```diff
@@ -0,0 +1,190 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines types used by the GA4GH Allele Annotation API.
+*/
+
+protocol AlleleAnnotations{
+
+import idl "common.avdl";
+import idl "metadata.avdl";
+import idl "variants.avdl";
+
+/*
+
+The VariantAnnotation record groups different types of annotation records by
+Variant.
+
+The TranscriptEffect sub record holds information on the effect of a specific
+allele on a specific transcript.
+As Variants may overlap multiple transcripts, they may have multiple
+TranscriptEffect records. Variants with multiple alternate alleles will have
+multiple TranscriptEffect records per transcript.
+(2 alternate alleles x 3 transcripts = 6 TranscriptEffect records)
+
+VariantAnnotation records belong to VariantAnnotationSets.
+VariantAnnotationSets are created by comparing a number of Variants from a
+VariantSet to a specific set of reference data using specific software tools.
+A VariantAnnotationSet contains information on reference data and software
+versions used in calculating the annotation; it is essential this information
+is exhaustive.
+
+*/
+
+/**
+An AnalysisResult record holds the output of a prediction package such
+as SIFT on a specific allele.
+*/
+record AnalysisResult {
+
+  /** The analysis record for this result (defined in metadata schema)  */
+  Analysis analysis;
+
+  /** The text-based result for this analysis */
+  union { null, string} analysisResult;
+
+  /** The numeric score for this analysis  */
+  union { null, int} analysisScore;
+
+}
+
+/**
+An allele location record holds the location of an allele relative to a
+non-genomic coordinate system such as a CDS or protein and holds the
+reference and alternate sequence where appropriate
+*/
+record AlleleLocation {
+
+  /** Relative start position of the allele in this coordinate system */
+  int overlapStart ;
```

Comment:
Why do we need the `overlap` prefix here? All other location references use `start` and `end`.